nipype/interfaces/fsl/epi.py

```diff
@@ -354,14 +354,14 @@ class ApplyTOPUP(FSLCommand):
 
     >>> from nipype.interfaces.fsl import ApplyTOPUP
     >>> applytopup = ApplyTOPUP()
-    >>> applytopup.inputs.in_files = ["epi.nii", "epi_rev.nii"]
+    >>> applytopup.inputs.in_files = ["ds003_sub-01_mc.nii.gz", "ds003_sub-01_mc.nii.gz"]
```

Comment:
Bc I added the check for dimensionality, empty files were causing the tests to fail. `ds003_sub-01_mc.nii.gz` is an already-existing file in `testing/data` that has the appropriate dimensions. It is a little hacky but consistent with the rest of nipype's testing infrastructure.